x-pack/plugins/security_solution/server/endpoint/services/artifacts/manifest_manager/manifest_manager.ts

```diff
@@ -372,16 +370,10 @@ export class ManifestManager {
 
     for (const result of results) {
       await iterateArtifactsBuildResult(result, async (artifact, policyId) => {
-        let artifactToAdd = baselineManifest.getArtifact(getArtifactId(artifact)) || artifact;
-
-        if (!isCompressed(artifactToAdd)) {
-          artifactToAdd = await maybeCompressArtifact(artifactToAdd);
-
-          if (!isCompressed(artifactToAdd)) {
-            throw new Error(`Unable to compress artifact: ${getArtifactId(artifactToAdd)}`);
-          } else if (!internalArtifactCompleteSchema.is(artifactToAdd)) {
-            throw new Error(`Incomplete artifact detected: ${getArtifactId(artifactToAdd)}`);
-          }
+        const artifactToAdd = baselineManifest.getArtifact(getArtifactId(artifact)) || artifact;
+        artifactToAdd.compressionAlgorithm = 'none';
```

Comment:
It needs to complain the schema (x-pack/plugins/security_solution/common/endpoint/schema/common.ts). 
As there will be existing artifacts with zip compression and others without it (newers) I think it's good to keep both types (zlib and none). Maybe I'm wrong, thoughts @paul-tavares ?

Reply:
I'm still reviewing the PR and want to check it out locally to run a few tests. I think we still need it because we keep an "internal" copy of the manifest entries as well as then use that data to populate the manifest entries that are added to the Policy.